File: round 1/Beast/problem1.java
#include<stdio.h>
int power(int a,int b)
{
long long int i,p=1;
for(i=1;i<b;i++)
p=p*a;
return p;
}
int main()
{
int n;
scanf("%d",&n);
if(n%2!=0)
printf("%d",n);
else
printf("%lld",power(3,(n/2-1)));
}

File: round 1/Beast/problem1.c
#include<stdio.h>
#include<math.h>
int main()
{
int n,flag=0,i;
scanf("%d",&n);
for(i=2;i<=sqrt(n);i++)
{
if(n%i==0)
{
flag=1;
break;
}
}
if(flag==0)
printf("Yes");
else
printf("No");
}